samtranslator/model/eventsources/push.py

```diff
@@ -614,6 +615,17 @@ def _add_swagger_integration(self, api, function):
                 editor.add_request_model_to_method(path=self.Path, method_name=self.Method,
                                                    request_model=self.RequestModel)
 
+        if self.ContentHandling:
+            content_handling = self.ContentHandling
+
+            if content_handling:
+                if content_handling not in ['CONVERT_TO_BINARY', 'CONVERT_TO_TEXT']:
+                    raise InvalidEventException(
+                        self.relative_id,
+                        'Unable to set ContentHandling, must be one of {contentHandlingList}'
+                        .format(contentHandlingList=['CONVERT_TO_BINARY', 'CONVERT_TO_TEXT']))
```

Comment:
We use this same list in multiple different places, could you import it from swagger. _CONTENT_HANDLING_ALLOWED_VALUES instead?

Reply:
done! thanks!